tools/cockpit.spec

```diff
@@ -738,13 +743,14 @@ bastion hosts, and a basic dashboard.
 
 %ifarch x86_64 %{arm} aarch64 ppc64le i686 s390x
 
+%if 0%{?fedora} || 0%{?rhel} < 8
 %package -n cockpit-docker
 Summary: Cockpit user interface for Docker containers
 Requires: cockpit-bridge >= %{required_base}
 Requires: cockpit-shell >= %{required_base}
 Requires: /usr/bin/docker
 Requires: /usr/lib/systemd/system/docker.service
-%if 0%{?fedora} || 0%{?rhel} >= 8
+%if 0%{?fedora}
```

Comment:
This can be dropped entirely (it was already redundant before this PR). Yay some simplification :-)

Reply:
We only use python 3 here?